src/main/webapp/resources/cart.html

```diff
@@ -0,0 +1,152 @@
+            <nav class="navbar navbar-dark bg-primary">
+                <h2>T-shop</h2>
+            </nav>
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="col">
+            <ul class="nav justify-content-center">
+                <li class="nav-item">
+                    <a class="nav-link active" href="index.html">Home</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#">Products</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="my-orders.html">My orders</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="cart.html">Cart</a>
+                </li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="row" id="product-table">
+        <div class="col-8">
+            <h4 id="cart-title">My order:</h4>
+            <table class="table table-borderless">
+                <thead>
+                <tr>
+                    <th scope="col"></th>
+                    <th scope="col"></th>
+                    <th scope="col"></th>
+                    <th scope="col"></th>
+                </tr>
+                </thead>
+                <tbody id="cart-items">
+                </tbody>
+            </table>
+        </div>
+        <div class="col-4" id="submit-order-card">
+            <div class="card">
+                <div class="card-body">
+                    <h5 class="card-title">Total price</h5>
+                    <h6 class="card-subtitle mb-2 text-muted" id="total-price"></h6>
+                    <a href="#" class="btn btn-primary" id="submit-order">Submit order</a>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="row" id="additional-info">
+        <div class="col">
+            <form>
+                <div class="form-row">
+                    <div class="form-group col-md-6">
+                        <label for="first-name">First name</label>
+                        <input type="text" class="form-control" id="first-name" placeholder="First name">
+                    </div>
+                    <div class="form-group col-md-6">
+                        <label for="last-name">Last name</label>
+                        <input type="text" class="form-control" id="last-name" placeholder="Last name">
+                    </div>
+                </div>
+                <div class="form-row">
+                    <div class="form-group col-md-6">
+                        <label for="inputPhone">Phone number</label>
+                        <input type="text" class="form-control" id="inputPhone" placeholder="Phone number">
+                    </div>
+                    <div class="form-group col-md-6">
+                        <label for="inputEmail4">Email</label>
+                        <input type="email" class="form-control" id="inputEmail4" placeholder="Email">
```

Comment:
Please rename the ID from `inputEmail4` into `inputEmail`. Usage of digits in identifiers is a bad practice.